
Thu, 04 Feb 2010 15:30:46 GMT

MAKE:
mv pidigits.python-4.python pidigits.python-4.py
/usr/bin/python -OO -c "from py_compile import compile; compile('pidigits.python-4.py')"
0.08s to complete and log all make actions

COMMAND LINE:
/usr/bin/python pidigits.python-4.pyo 2000

PROGRAM FAILED 


PROGRAM OUTPUT:

Traceback (most recent call last):
  File "pidigits.python-4.py", line 7, in <module>
    import gmpy
ImportError: No module named gmpy
